lib/services/insights/README.md

```diff
@@ -23,13 +23,8 @@ npm install azure-arm-insights
 ### Create the InsightsManagementClient
 
 ```javascript
-var common               = require("azure-common"),
-    insightsClientLib    = require("azure-arm-insights");
-
-var insightsManagementClient = insightsClientLib.createInsightsManagementClient(new common.TokenCloudCredentials({
-  subscriptionId: "<your subscription id>",
-  token: "<your token here>"
-}));
+var common               = require("ms-rest-azure"),
```

Comment:
`var msrestAzure`. Please take a look at the [readme of storageManagement](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/storageManagement2#how-to-use) as an example and model things accordingly for insights.